samplers/jaegerremote/sampler_remote.go

```diff
@@ -95,11 +95,27 @@ func New(
 	return sampler
 }
 
+
+// Deprecated: mutex is internal and shall not be used.
```

Comment:
Since this is an experimental version, this section has been temporarily marked for now, pending feedback from other approvers.